docs/src/layouts/links.js

```diff
@@ -38,6 +38,12 @@ const ComponentsLinks = [
         children: 'Bottom sheet',
         tags: ['ios'],
       },
+      {
+        id: 'BOTTOM_NAV',
```

Comment:
Can this be moved to before the BOTTOM_SHEET for alphabetical order?

Reply:
Note it automatically orders them on the site, so this is a minor thing